jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/extensions/WebSocketExtensionFactory.java

```diff
@@ -42,10 +37,8 @@
 
 public class WebSocketExtensionFactory extends ExtensionFactory implements LifeCycle, Dumpable
```

Comment:
This class extends a deprecated class -- should this class be deprecated as well?